cmd/dockerd/bin_windows_arm64.go

```diff
@@ -0,0 +1,8 @@
+//go:build windows && arm64
+// +build windows,arm64
+
+//go:generate goversioninfo -arm=true -64=true -o=../../cli/winresources/dockerd/resource.syso -icon=winresources/dockerd.ico -manifest=winresources/dockerd.exe.manifest ../../cli/winresources/dockerd/versioninfo.json
+
+package main
+
+import _ "github.com/docker/docker/cli/winresources/dockerd"
```

Comment:
I'd suggest maybe `bin` is too generic a name for this file. What about `genversioninfo_windows_arm64.go` and friends? Something that gives a hint what it does?

Reply:
sure sgtm